src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+import type {ElementRef} from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import checkboxStyles from './checkbox.scss';
+
+type Props = {
+    checked: boolean,
```

Comment:
Yeah, checkbox also deliver the value if clicked, but that's something I never really got, IMO a boolean value represents the state of a checkbox a lot better.

And if we add the value as well it makes the component more complicated, do you think that's worth it?